src/mmw/js/src/modeling/controls.js

```diff
@@ -111,8 +111,22 @@ var InputInfoView = Marionette.ItemView.extend({
     },
 
     templateHelpers: function() {
+        var output = this.model.get('output'),
+            errorMessage = output && output.errorMessages[this.userInputName],
+            infoMessage = output && output.infoMessages[this.userInputName],
```

Comment:
I did not know that using `&&` in this way would return the correct value here. Pretty interesting.